vgen/src/main/java/io/gapi/vgen/py/PythonLanguageProvider.java

```diff
@@ -58,9 +58,10 @@
 
     Doc filenameDoc = snippets.generateFilename(element);
     String outputFilename = filenameDoc.prettyPrint();
+    Doc body = snippets.generateBody(element);
     List<String> importList = importHandler.calculateImports();
     // Generate result.
-    Doc result = snippets.generateClass(element, importList);
+    Doc result = snippets.generateModule(element, body, importList);
```

Comment:
The old process for Gapic library codegen is preserved: the corresponding generateBody snippet is degenerate. But we needed snippet-dependent imports to support proper sample gen for discovery (and, presumably, eventually for Gapic), so I changed the snippet interface to be like that for Java.